phylogenetic/rules/annotate_phylogeny.smk

```diff
@@ -45,6 +45,11 @@ rule ancestral:
         "benchmarks/{build}/ancestral.txt",
     params:
         inference = config["ancestral"]["inference"],
+        root_sequence=lambda w: (
+            ("--root-sequence " + resolve_config_path(config["ancestral"]["root_sequence"])(w))
+            if config.get("ancestral").get("root_sequence")
```

Comment:
The `ancestral` param is already required [above](https://github.com/nextstrain/mumps/blob/7963046977ef815e6077466523fe4c7ba54eb1f9/phylogenetic/rules/annotate_phylogeny.smk#L47), so it can be accessed directly

```suggestion
            if config["ancestral"].get("root_sequence")
```

If you wanted to make `ancestral` fully optional, then provide an empty dict as the default value to prevent errors on the second `.get` call

```suggestion
            if config.get("ancestral", {}).get("root_sequence")
```